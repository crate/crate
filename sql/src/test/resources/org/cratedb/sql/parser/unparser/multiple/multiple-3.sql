CREATE TABLE t1(id INT PRIMARY KEY, name VARCHAR(10) NOT NULL); INSERT INTO t1 VALUES(1, 'foo'); INSERT INTO t1 VALUES(2, 'bar');
