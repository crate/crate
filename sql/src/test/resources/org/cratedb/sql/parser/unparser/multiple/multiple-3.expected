[0]: CREATE TABLE t1(PRIMARY KEY(id), id INTEGER, name VARCHAR(10) NOT NULL);
[1]: INSERT INTO t1 VALUES(1, 'foo');
[2]: INSERT INTO t1 VALUES(2, 'bar');